scripts/assets/calico-allow-policy.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: allow-apps
+spec:
+  namespaceSelector: has(korifi.cloudfoundry.org/space-guid)
+  types:
+  - Ingress
+  - Egress
+  ingress:
```

Comment:
Security groups are meant to configure apps egress (outbound traffic) but not ingress. Is it necessary to configure ingress here?